tests/lib/filterAtfs.test.ts

```diff
@@ -0,0 +1,66 @@
+import { AuthorisedTestingFacility } from '../../src/models/authorisedTestingFacility';
+import { filterAtfs } from '../../src/lib/filterAtfs';
+
+describe('filterAtfs library unit tests', () => {
```

Comment:
Please can you add some tests for dodgy or missing data? Example, if the endDate is missing or an invalid date etc.

Reply:
Good point! I've now added a check in my latest commit that will validate the end date before checking if its up to date availability information or not. 
Any ATFs with invalid end dates I consider to be part of the 'No Information' group, so these aren't filtered out regardless of the isAvailable property